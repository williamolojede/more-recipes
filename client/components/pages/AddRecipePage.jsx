import React from 'react';

import ManageRecipes from '../shared/ManageRecipes';
import AddRecipeForm from '../shared/AddRecipeForm';

const AddRecipePage = props => (
  <ManageRecipes>
    <AddRecipeForm isEditMode={false} {...props} />
  </ManageRecipes>
);
export default AddRecipePage;
