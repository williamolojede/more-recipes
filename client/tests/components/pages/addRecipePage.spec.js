import React from 'react';
import AddRecipePage from '../../../components/pages/AddRecipePage';

function setup() {
  const wrapper = shallow(<AddRecipePage />);

  return {
    wrapper
  };
}

describe('Add Recipe Page', () => {
  it('should render with right amount of elements', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
