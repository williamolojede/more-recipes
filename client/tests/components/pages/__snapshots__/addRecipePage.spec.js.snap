// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Recipe Page should render with right amount of elements 1`] = `
<Connect(ManageRecipes)>
  <Connect(AddRecipeForm)
    isEditMode={false}
  />
</Connect(ManageRecipes)>
`;
